### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by gsi-kevincarrera** [incluud/accessible-astro-components#75]
I believe it is easier to understand the library through a documentation, also extends the possibility of providing more information.

**Comment by markteekman** [comment]
@gsi-kevincarrera yes, this is something I've been wanting for a long time now and I even made a start about two years ago. It would also be a great bridge between using the Accessible Astro Starter theme in collaboration with the Accessible Astro Components package. But because of time constraints I never got to finishing it and this continues to be the situation.

For now the README.md should suffice and maybe somewhere in the future I can pick up https://github.com/markteekman/accessible-astro-docs and maybe update it using the new Starfield theme (https://starlight.astro.build/), granted it is fully accessible of course 🙂

**Comment by gsi-kevincarrera** [comment]
Im open to contribute on this project, with documentation or even testing. I really like it :)

**Comment by markteekman** [comment]
@gsi-kevincarrera that's really great to hear! And it's very much appreciated 😄 I've set up a Netlify deploy for the repository so your good to go: https://accessible-astro-docs.netlify.app/ (failed deploy for now but that should be fixed by updating the project)! We could break it down into a couple of steps:

- [ ] Clean-up repository and (de)install dependencies
- [ ] Setup the https://starlight.astro.build/ theme
- [ ] Setup sections (Getting started / Accessibility tips / Themes / Components / Contributions)
- [ ] Write content for each section, using the README.md for the components
- [ ] Audit the Starlight theme and check for potential accessibility issues

What are your thoughts on it? 🙂

**Comment by markteekman** [comment]
Closed in #96
